.github/workflows/python-package-conda.yml

```diff
@@ -61,5 +61,6 @@ jobs:
       - name: Test with pytest
         run: |
           conda activate genomics.py3
+          export PYTHONPATH=$(pwd)
```

Comment:
So that jupyter will be able to import ugvc

Reply:
It does not quite make sense, ugvc is installed in the environment

Reply:
1. Where is it installed? I don't see it installed in the workflow.
    It is added to the path by pytest, but this is not enough for jupyter.
    
    Here you can see the test passes (since I added ugvc to path with PYTHONPATH), and later it fail after I remove it from path:
    
![image](https://user-images.githubusercontent.com/44941371/219455036-638d39f1-28e5-4e64-9653-0c83d4fa4e0e.png)

2. Another question is why do we need to export PYTHONPATH when running regular jupyter convert with installed ugvc. When I think of installed I think of "cd VariantCalling; pip install ."

Here you can see the failure of the previous jupyter convert command, and then when I add to path it completes:
![image](https://user-images.githubusercontent.com/44941371/219455723-c5ac16e6-dcd7-40d1-b8a0-a7ed6e82a49f.png)

Reply:
I have just checked, and papermill doesn't solve this

Reply:
I miss something? On my laptop I don't need this
(genomics.py3) ilyas@ilya-laptop:~/proj/BioinfoResearch/VariantCalling$ echo $PYTHONPATH


(genomics.py3) ilyas@ilya-laptop:~/proj/BioinfoResearch/VariantCalling$ pytest test/system/test_vc_report.py 
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.7.8, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /home/ilyas/proj/BioinfoResearch/VariantCalling
plugins: cov-3.0.0, mock-3.6.1, xdist-2.3.0, forked-1.3.0
collected 4 items                                                                                                                                                                                                 

test/system/test_vc_report.py ..

Reply:
oh - maybe __init__.py was missing in the reports?

Reply:
ok, fixed